pywbem/_logging.py

```diff
@@ -14,404 +14,341 @@
 # License along with this program; if not, write to the Free Software
 # Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
 
+# pylint: disable=line-too-long
 """
-**Experimental:** *New in pywbem 0.11 as experimental.*
+**Experimental:** *Configuration methods completely rewritten in pywbem 0.12.*
```

Comment:
I suggest to still mention the original introduction, e.g. "New in pywbem 0.11, redesigned in pywbem 0.12 as experimental"